src/main/java/ca/uhn/fhir/jpa/starter/HapiProperties.java

```diff
@@ -42,6 +42,7 @@
     static final String TESTER_CONFIG_REFUSE_TO_FETCH_THIRD_PARTY_URLS = "tester.config.refuse_to_fetch_third_party_urls";
     static final String CORS_ENABLED = "cors.enabled";
     static final String CORS_ALLOWED_ORIGIN = "cors.allowed_origin";
+    static final String CORS_ALLOWED_CREDENTIALS = "hapi.properties";
```

Comment:
I guess this should probably be `CORS_ALLOW_CREDENTIALS` (no 'ed') and I think the value is wrong :)

Reply:
you are right. Stupid minor mistakes. Fixed it